lib/grist-api.ts

```diff
@@ -30,17 +34,17 @@ export async function getAPIKey(): Promise<string> {
   }
 
   // Otherwise, assume we are in node environment.
-  if (process.env.GRIST_API_KEY) {
+  if (process.env.GRIST_API_KEY !== undefined) {
```

Comment:
Oh I see, so you set it to blank in the environment, so that the code doesn't reach the file system check? Makes some sense.